lib/HTFeed/Storage.pm

```diff
@@ -300,26 +331,37 @@ sub move {
     my $self = shift;
 
     get_logger->debug("  starting move");
+    my $start_time  = $self->{job_metrics}->time;
     my $volume      = $self->{volume};
     my $mets_stage  = $self->mets_stage_path;
     my $zip_stage   = $self->zip_stage_path;
     my $object_path = $self->object_path;
 
     # make sure the operation will succeed
     if (-f $mets_stage and -f $zip_stage and -d $object_path) {
-	$self->safe_system('mv', '-f', $mets_stage, $object_path);
-	$self->safe_system('mv', '-f', $zip_stage,  $object_path);
-	get_logger->debug("  finished move");
-	return 1;
+        my $bytes_moved = (-s $mets_stage || 0) + (-s $zip_stage || 0);
+        $self->safe_system('mv', '-f', $mets_stage, $object_path);
+        $self->safe_system('mv', '-f', $zip_stage,  $object_path);
+
+        get_logger->debug("  finished move");
+        my $end_time    = $self->{job_metrics}->time;
+        my $delta_time  = $end_time - $start_time;
+        $self->{job_metrics}->inc("ingest_move_items");
+        $self->{job_metrics}->add("ingest_move_seconds", $delta_time);
+        $self->{job_metrics}->add("ingest_move_bytes_read", $bytes_moved);
```

Comment:
This may not be super relevant since the staging path should typically be on the same filesystem as the object path, so the move becomes an atomic rename rather than a copy that requires reading/writing the items again..

Reply:
We should also review the subclasses (especially `HTFeed::Storage::ObjectStore` since there are some different implementations that may override the metrics recording here.

Reply:
No longer measuring `sub move`: https://github.com/hathitrust/feed/pull/128/commits/7f35bc7cfe702d0a168ac0b9be4f01f2b1a8b55e

Reply:
Made issue #136 for `HTFeed::Storage::ObjectStore`.